packages/tracing/src/browser/metrics.ts

```diff
@@ -194,60 +194,11 @@ export class MetricsInstrumentation {
       }
 
       transaction.setMeasurements(this._measurements);
-      this._tagMetricInfo(transaction);
-
+      _tagMetricInfo(transaction, this._lcpEntry, this._clsEntry);
```

Comment:
I believe we dont use `_` for regular functions. Just private class members due to linting/mangling setup.

Reply:
Yeah this is just there due to copy paste, I'll change.